gammagl/models/grade.py

```diff
@@ -0,0 +1,103 @@
+import tensorlayerx as tlx
+from gammagl.layers.conv import GCNConv
+import numpy as np
+
+
+# Multi-layer Graph Convolutional Networks
+class GCN(tlx.nn.Module):
+    def __init__(self,in_dim, out_dim,act_fn, num_layers = 2):
+        super(GCN, self).__init__()
+        assert num_layers>=2
+        self.num_layers=num_layers
+        self.convs=tlx.nn.ModuleList()
+
+        self.convs.append(GCNConv(in_dim,out_dim*2))
+        for _ in range(self.num_layers-2):
+            self.convs.append(GCNConv(out_dim*2,out_dim*2))
+
+        self.convs.append(GCNConv(out_dim*2,out_dim))
+        self.act_fn=act_fn
+
+
+    def forward(self, feat,edge_index):
+        for i in range(self.num_layers):
+            feat=self.act_fn(self.convs[i](feat,edge_index))
+        return feat
+
+# Multi-layer(2-layer) Perceptron
+class MLP(tlx.nn.Module):
+    def __init__(self, in_feat, out_feat):
+        super(MLP, self).__init__()
+        # init = tlx.nn.initializers.HeNormal(a=math.sqrt(5))
+        self.fc1 = tlx.nn.Linear(in_features=in_feat, out_features=out_feat) #, W_init=init)
+        self.fc2 = tlx.nn.Linear(in_features=out_feat, out_features=in_feat) # , W_init=init)
+
+    def forward(self, x):
+        x = tlx.elu(self.fc1(x))
+        return self.fc2(x)
+
+class GRADE(tlx.nn.Module):
+    r"""
```

Comment:
Remember to add more detailed rst document. The reference is [here](https://docs.qq.com/pdf/DUXRTTU9tUnB1WnFB?).

Reply:
I have added more details, please check my changes.